cabby/data/extract_wikigeo_contexts.py

```diff
@@ -11,12 +11,16 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+'''Command line method to extract Wikipedia and Wikidata items and save to file.
```

Comment:
And OSM:
Command line method to extract Wikipedia, Wikidata, and OSM items, and construct WikiGeo samples and save to file.

Reply:
Fixed, thanks!